kong/plugins/aws-lambda/handler.lua

```diff
@@ -51,6 +52,17 @@ function AWSLambdaHandler:access(conf)
     query = conf.qualifier and "Qualifier=" .. conf.qualifier
   }
 
+  if conf.aws_iam_role then
```

Comment:
@udangel-r7 This will change the default behaviour if contributed back to the open source plugin — I'm trying to keep this optional.